### PR TITLE
Import HttpUtilities and MediaUtilities from the SOAP library. Only r…

### PR DESCRIPTION
**Pull request by AnashOommen**
…equired methods are imported, the rest have been removed for now.

### CODE REVIEW

File: src/Util/MediaUtilities.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            using (Stream responseStream = response.GetResponseStream())
+            {
+                responseStream.CopyTo(memStream);
+            }
+
+            return memStream.ToArray();
+        }
+
+        /// <summary>
+        /// Retrieves an asset from the web given its url.
```
Comment (sundquist) [inline src/Util/MediaUtilities.cs]: Nit: url => URL

File: src/Util/MediaUtilities.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        /// </summary>
+        /// <param name="assetUrl">The url of the asset to be retrieved.</param>
```
Comment (sundquist) [inline src/Util/MediaUtilities.cs]: Nit: url => URL

File: src/Util/MediaUtilities.cs (hunk 1 above)
Comment (sundquist) [inline src/Util/MediaUtilities.cs]: Nit: url => URL

File: tests/Util/MediaUtilitiesTest.cs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        /// Tears down this instance.
+        /// </summary>
+        [TearDown]
+        public void Teardown()
+        {
+            File.Delete(fileUri.AbsolutePath);
+        }
+
+        /// <summary>
+        /// Tests if data can be retrieved from a url.
```
Comment (sundquist) [inline tests/Util/MediaUtilitiesTest.cs]: Nit: url => URL

File: tests/Util/MediaUtilitiesTest.cs [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        [Test]
+        [Category("Small")]
+        public void TestGetAssetDataFromUrl1()
+        {
+            byte[] data = MediaUtilities.GetAssetDataFromUrl(fileUri, new GoogleAdsConfig());
+            Assert.AreEqual(FILE_CONTENTS, Encoding.UTF8.GetString(data));
+        }
+
+        /// <summary>
+        /// Tests if data can be retrieved from a url.
```
Comment (sundquist) [inline tests/Util/MediaUtilitiesTest.cs]: Nit: url => URL

File: src/Util/MediaUtilities.cs [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+namespace Google.Ads.GoogleAds.Util
+{
+    /// <summary>
+    /// Provides utility methods for handling media resources.
+    /// </summary>
+    public static class MediaUtilities
+    {
+        /// <summary>
+        /// Retrieves an asset from the web given its url.
```
Comment (sundquist) [inline src/Util/MediaUtilities.cs]: Nit: url => URL

File: src/Util/MediaUtilities.cs [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+        /// </summary>
+        /// <param name="assetUrl">The url of the asset to be retrieved.</param>
```
Comment (sundquist) [inline src/Util/MediaUtilities.cs]: Nit: url => URL

File: src/Util/MediaUtilities.cs (hunk 5 above)
Comment (AnashOommen) [reply src/Util/MediaUtilities.cs]: Done

File: src/Util/MediaUtilities.cs (hunk 6 above)
Comment (AnashOommen) [reply src/Util/MediaUtilities.cs]: Done

File: src/Util/MediaUtilities.cs (hunk 1 above)
Comment (AnashOommen) [reply src/Util/MediaUtilities.cs]: Done

File: src/Util/MediaUtilities.cs (hunk 2 above)
Comment (AnashOommen) [reply src/Util/MediaUtilities.cs]: Done

File: tests/Util/MediaUtilitiesTest.cs (hunk 3 above)
Comment (AnashOommen) [reply tests/Util/MediaUtilitiesTest.cs]: Done

File: tests/Util/MediaUtilitiesTest.cs (hunk 4 above)
Comment (AnashOommen) [reply tests/Util/MediaUtilitiesTest.cs]: Done

**Pull request merged**